core/config/docs/chains-solana.toml

```diff
@@ -95,6 +95,22 @@ FinalityTagEnabled = true # Default
 # FinalizedBlockOffset is the offset from the finalized block to use for finality tags.
 FinalizedBlockOffset = 0 # Default
 
+[Solana.Workflow]
+# AcceptanceTimeout is the default timeout for a tranmission to be accepted on chain
+AcceptanceTimeout = '45s' # Default
+# PollPeriod is the default poll period for checking transmission state
+PollPeriod = '3s' # Default
+# ForwarderAddress is the keystone forwarder program address on chain.
+ForwarderAddress = '14grJpemFaf88c8tiVb77W7TYg2W3ir6pfkKz3YjhhZ5' # Example
+# ForwarderState is the keystone forwarder program state account on chain.
+ForwarderState = '14grJpemFaf88c8tiVb77W7TYg2W3ir6pfkKz3YjhhZ5' # Example
+# FromAddress is Address of the transmitter key to use for workflow writes.
+FromAddress = '4BJXYkfvg37zEmBbsacZjeQDpTNx91KppxFJxRqrz48e' # Example
+# GasLimitDefault is the default gas limit for workflow transactions.
+GasLimitDefault = 300_000 # Default
+# TxAcceptanceState is the default acceptance state for writer DON tranmissions.
+TxAcceptanceState = 3 # Default
```

Comment:
Please make sure that all of the new fields are documented. You may have bypassed the check by using non-pointer fields.